core/src/test/java/com/linecorp/armeria/client/HttpClientUnwrapTest.java

```diff
@@ -32,7 +32,7 @@ void test() {
         final WebClient client =
                 WebClient.builder()
                          .decorator(LoggingClient.newDecorator())
-                         .decorator(RetryingClient.newDecorator(RetryStrategy.never()))
+                         .decorator(RetryingClient.newDecorator(RetryRule.builder().thenNoRetry()))
```

Comment:
Question - Should we add `RetryRule.never()`?

Reply:
`RetryRule.never()` seems not to use often, so removed it from factories. 
I thought that if a user doesn't want to retry failed requests, he/she can just remove `RetryClient` from decorators.